LICENSE

```diff
@@ -432,3 +432,11 @@ The modern-go/concurrent component is used under the Apache 2.0 license:
 The modern-go/reflect2 component is used under the Apache 2.0 license:
 @vendor/github.com/modern-go/reflect2/*
 ./vendor/github.com/modern-go/reflect2/LICENSE
+
+For the lestrrat-go/jwx component:
+@traffic_ops/traffic_ops_golang/vendor/github.com/lestrrat-go/jwx/*
```

Comment:
I'm not sure if this is the right place, but it'd be nice to make a note somewhere of the git hash at which the repository was vendored. Ideally they have versioned releases, so you can just say that, but if you cloned them from master, then we probably want to know the actual hash.

Reply:
done